cdap-ui/app/tracker/controllers/container-ctrl.js

```diff
@@ -28,7 +28,5 @@ class TrackerContainerController {
   }
 }
 
-TrackerContainerController.$inject = ['$state'];
```

Comment:
Just to confirm, we don't need this anymore because of ngInject, right?

Reply:
yup